components/create_post/create_post.jsx

```diff
@@ -575,7 +579,9 @@ export default class CreatePost extends React.Component {
             return;
         }
 
-        if (!e.ctrlKey && !e.metaKey && !e.altKey && !e.shiftKey && e.keyCode === KeyCodes.UP && this.state.message === '') {
+        const editLastPostKey = !e.ctrlKey && !e.metaKey && !e.altKey && !e.shiftKey && e.keyCode === KeyCodes.UP;
+        const replyToLastPostKey = !e.ctrlKey && !e.metaKey && !e.altKey && e.shiftKey && e.keyCode === KeyCodes.UP;
+        if (editLastPostKey && this.state.message === '') {
```

Comment:
Can you replace `this.state.message` with `textboxUserInputEmpty` in this check? Otherwise, it looks like you aren't using that new state variable that you introduced for this very purpose.

Reply:
That makes sense, thanks @MusikPolice!
PR updated.